app/js/lib/view_handler/enabled_when_valid.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+var objectAsssign = require( 'object-assign' ),
+	EnabledWhenValidHandler = {
+		elements: null,
+		update: function ( state ) {
+			if ( typeof state  !== 'undefined' && state.isValid && state.isValidated ) {
```

Comment:
I don't think you need to check here whether the state is `undefined`. Because usually, you will invoke the root reducer at least once, and then all the subreducers are invoked and initialise the state (since you always handle `typeof state === 'undefined'` in the reducers)